recipes-bsp/rpi-boot-setting/rpi-u-boot-scr.bb

```diff
@@ -14,17 +14,22 @@ inherit dpkg
 
 DEBIAN_BUILD_DEPENDS = "u-boot-tools"
 
-SRC_URI = "file://debian/ file://boot.cmd"
+SRC_URI = "file://debian/"
+SRC_URI += " file://raspberrypi3bplus-64_boot.cmd"
```

Comment:
Change this line as below.

`SRC_URI:raspberrypi3bplus-64 += " file://raspberrypi3bplus-64_boot.cmd"`

Reply:
I fixed it as you mentioned.